menu/freedesktop/asebachallenge.desktop

```diff
@@ -9,5 +9,5 @@ GenericName[es]=Aseba simulador de robot para jugar en la red
 GenericName[pt]=Aseba simulador de robô para jogar em rede
 Exec=asebachallenge
 TryExec=asebachallenge
-Categories=Education;Science;Aseba
+Categories=Education;Science;Aseba;
```

Comment:
Yes, but it's not critical. During the package build process I just get warnings like this:

```
[  346s] + desktop-file-install --remove-category=Aseba --dir=/home/abuild/rpmbuild/BUILDROOT/aseba-1.4.0-0.2.20150912git04b42d4.fc22.x86_64/usr/share/applications /home/abuild/rpmbuild/BUILDROOT/aseba-1.4.0-0.2.20150912git04b42d4.fc22.x86_64/usr/share/applications/asebachallenge.desktop /home/abuild/rpmbuild/BUILDROOT/aseba-1.4.0-0.2.20150912git04b42d4.fc22.x86_64/usr/share/applications/asebaonlinehelp.desktop /home/abuild/rpmbuild/BUILDROOT/aseba-1.4.0-0.2.20150912git04b42d4.fc22.x86_64/usr/share/applications/asebaplayground.desktop /home/abuild/rpmbuild/BUILDROOT/aseba-1.4.0-0.2.20150912git04b42d4.fc22.x86_64/usr/share/applications/asebastudio.desktop /home/abuild/rpmbuild/BUILDROOT/aseba-1.4.0-0.2.20150912git04b42d4.fc22.x86_64/usr/share/applications/asebastudiothymio.desktop /home/abuild/rpmbuild/BUILDROOT/aseba-1.4.0-0.2.20150912git04b42d4.fc22.x86_64/usr/share/applications/thymioupgrader.desktop /home/abuild/rpmbuild/BUILDROOT/aseba-1.4.0-0.2.20150912git04b42d4.fc22.x86_64/usr/share/applications/thymiovpl.desktop
[  346s] /home/abuild/rpmbuild/BUILDROOT/aseba-1.4.0-0.2.20150912git04b42d4.fc22.x86_64/usr/share/applications/asebaplayground.desktop: warning: key "Categories" is a list and does not have a semicolon as trailing character, fixing
[  346s] /home/abuild/rpmbuild/BUILDROOT/aseba-1.4.0-0.2.20150912git04b42d4.fc22.x86_64/usr/share/applications/asebastudio.desktop: warning: key "Categories" is a list and does not have a semicolon as trailing character, fixing
[  346s] /home/abuild/rpmbuild/BUILDROOT/aseba-1.4.0-0.2.20150912git04b42d4.fc22.x86_64/usr/share/applications/asebastudiothymio.desktop: warning: key "Categories" is a list and does not have a semicolon as trailing character, fixing
[  346s] /home/abuild/rpmbuild/BUILDROOT/aseba-1.4.0-0.2.20150912git04b42d4.fc22.x86_64/usr/share/applications/thymioupgrader.desktop: warning: key "Categories" is a list and does not have a semicolon as trailing character, fixing
[  346s] /home/abuild/rpmbuild/BUILDROOT/aseba-1.4.0-0.2.20150912git04b42d4.fc22.x86_64/usr/share/applications/thymiovpl.desktop: warning: key "Categories" is a list and does not have a semicolon as trailing character, fixing
```
If you want me to do another round of edits to the desktop files, I will. But if I do, I'm thinking we should also remove the "Aseba" category itself. I'm not sure why it's there, but the spec file runs desktop-file-install with --remove-category=Aseba so it's being removed during RPM packaging. Also, if you run desktop-file-validate on the desktop files, it complains that Aseba is not a standard category.

So, should I edit all the desktop files' Categories to remove "Aseba" and add a trailing ";" if necessary? Or do you just want to stick with it as is since the packages all built successfully)?